docs/hotkeys.md

```diff
@@ -0,0 +1,130 @@
+
+## Capture Button
+
+This hotkey emulates a press of the <hotkey v-bind:buttons='["A2"]'></hotkey> button as not all controllers may have this button natively on the controller.
+
+## SOCD Up Priority
+
+This hotkey changes the SOCD cleaning method to resolve to a neutral input (no input) on the X-axis, but prioritize the `Up` input on the Y-axis when both directions are pressed simultaneously. 
+
+| 1st Input + 2nd Input |  Result |
+|:---------------------:|:-------:|
+|      Left + Right     | Neutral |
+|      Right + Left     | Neutral |
+|       Up + Down       |    Up   |
+|       Down + Up       |    Up   |
+
+## SOCD Neutral
+
+This hotkey changes the SOCD cleaning method to resolve to a neutral input (no input) on both the X-axis and Y-axis when both directions are pressed simultaneously. 
+
+| 1st Input + 2nd Input |  Result |
+|:---------------------:|:-------:|
+|      Left + Right     | Neutral |
+|      Right + Left     | Neutral |
+|       Up + Down       | Neutral |
+|       Down + Up       | Neutral |
+
+## SOCD Last Win
+
+This hotkey changes the SOCD cleaning method to prioritize the second directional input on both the X-axis and Y-axis when both directions are pressed simultaneously. 
+
+| 1st Input + 2nd Input | Result |
+|:---------------------:|:------:|
+|      Left + Right     |  Right |
+|      Right + Left     |  Left  |
+|       Up + Down       |  Down  |
+|       Down + Up       |   Up   |
+
+## SOCD First Wins
+
+This hotkey changes the SOCD cleaning method to prioritize the first directional input on both the X-axis and Y-axis when both directions are pressed simultaneously. 
+
+| 1st Input + 2nd Input | Result |
+|:---------------------:|:------:|
+|      Left + Right     |  Left  |
+|      Right + Left     |  Right |
+|       Up + Down       |   Up   |
+|       Down + Up       |  Down  |
+
+## SOCD Cleaning Off
+
+This hotkey changes the SOCD cleaning method to not resolve any directional inputs on both the X-axis and Y-axis when both directions are pressed simultaneously. The controller will send both directional inputs. 
+
+?> This cleaning mode is only available in XInput mode as PS3/DirectInput, PS4 and Nintendo Switch modes do not support setting SOCD Cleaning to Off and will default to Neutral SOCD Cleaning mode.
+
+| 1st Input + 2nd Input |     Result     |
+|:---------------------:|:--------------:|
+|      Left + Right     | Left and Right |
+|      Right + Left     | Left and Right |
+|       Up + Down       |   Up and Down  |
+|       Down + Up       |   Up and Down  |
+
+## Invert X Axis
+
+This hotkey will invert the X-axis of your controller (i.e. pressing the Right button will result a Left input and vice versa).
+
+## Invert Y Axis
+
+This hotkey will invert the Y-axis of your controller (i.e. pressing the Up button will result a Down input and vice versa).
```

Comment:
"pressing the Up button will result a Down input"
I think a preposition is missing here, "will result in a"